exchange/bitswap/workers.go

```diff
@@ -9,16 +11,31 @@ import (
 	u "github.com/ipfs/go-ipfs/util"
 )
 
+var TaskWorkerCount = 16
+
+func init() {
+	twc := os.Getenv("IPFS_TASK_WORKERS")
```

Comment:
@whyrusleeping we may have other task workers. the problem with these env vars is that if we change them later we break user space. `IPFS_TASK_WORKERS` is way too general.